packages/carousel/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['@babel/env', '@babel/react']
+}
```

Comment:
Oh, just noticed the slider package has this too, but take a look at the `babel.config.js` in core and icons, which I think is the pattern we should go with for now